plenoptic/simulate/canonical_computations/steerable_pyramid_freq.py

```diff
@@ -306,7 +305,8 @@ def forward(self, x, scales=[]):
             If downsample is true, representation is an OrderedDict of the coefficients.
 
         """
-        self.pyr_coeffs = OrderedDict()
+        pyr_coeffs = OrderedDict()
+        self.pyr_size = OrderedDict()
```

Comment:
well the self.pyr_coeffs --> pyr_coeffs is because we dont want to store the coeffs as an attribute anymore for memory reasons. the self.pyr_size was more for readability since it always gets filled in the forward, might as well put it there, so it's close to where it's actually used, but I guess we can put it back in the initialization to keep all attributes there.